ha/core/config/config_manager.py

```diff
@@ -41,7 +41,7 @@ class ConfigManager:
     @staticmethod
     def init(log_name, log_path=None, level="INFO",
             backup_count=5, file_size_in_mb=10, syslog_server=None,
-            syslog_port=None, console_output=False, config_file=None):
+            syslog_port=None, console_output=True, config_file=None):
```

Comment:
Add a comment here mentioning that logs will always go to conole + to the log files

Reply:
Updated